README.md

```diff
@@ -5,6 +5,15 @@ The public facing website for rust dublin
 In order to build this site, run the follow commands in a terminal.
 Pull requests should be made to the public branch
 
+## pipeline
```

Comment:
In general the site is generated by processing the `src` input folder defined in `.eleventy.js` and outputting it to the `build` directory
2. It takes the `.md` files in the `src` folder merges them with page.njk and places the output in the `build` directory

Remove the reference to ~~using the template provided~~
~~this site outputs to build - this is controlled by .eleventy.js.~~